src/Core/Enums/LocalizationDirection.cs

```diff
@@ -14,4 +14,17 @@ public enum LocalizationDirection
     /// Right to left.
     /// </summary>
     rtl
+}
+
+public static class LocalizationDirectionExtension
+{
+    public static bool IsRTL(this LocalizationDirection direction)
+    {
+        return direction is LocalizationDirection.rtl;
+    }
+
+    public static LocalizationDirection GetDirectionFromBoolean(this bool boolValue)
```

Comment:
It's not a good idea to add an extension to a system type (bool): all uses of boolean will display this method.